src/fsharp/FSharp.Core.nuget/FSharp.Core.4.1.xxx.nuspec

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <!-- rename to FSharp.Core when done -->
+        <id>FSharp.Core</id>
+        <description>
+            FSharp.Core library - FSharp.Core.dll
+            Supported Platforms:
+              .NET 4.0
+              .NET Standard 1.6 (netstandard1.6)
```

Comment:
Looks like these comments are the wrong way around (4.1.xx has comments for 4.2.xx and vice versa)